### ISSUE TITLE
Search within "Discover"

### ISSUE THREAD

**Issue by azumukupoe** [Radarr/Radarr#5563]
**Is your feature request related to a problem? Please describe.**
I don't understand why I have to create a filter each time I want to search a movie within lists in "Discover"
Can't even create title filters to search by title

**Describe the solution you'd like**
Search within "Discover" by titles and filters that are already available
Should be able to search with filters in the main view as well
Such wasted potential...

~~**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.~~

~~**Additional context**
Add any other context or screenshots about the feature request here.~~

**Comment by bakerboy448** [comment]
>I don't understand why I have to create a filter each time I want to search a movie within lists in "Discover"
I'm not sure what you're asking?

>Can't even create title filters to search by title
correct

>Search within "Discover" by titles and filters that are already available
>Should be able to search with filters in the main view as well
?????

So are you asking to be able to filter by movie name in both the Index and Discover?

**Comment by azumukupoe** [comment]
> So are you asking to be able to filter by movie name in both the Index and Discover?

No, I do not want to create a filter each time I want to search for a movie
I want a quick filter, so that you don't have to create a filter every time

**Comment by bakerboy448** [comment]
So you mean filter like an Excel or Google Sheet filter where it's all inline with the table?

**Comment by azumukupoe** [comment]
I guess?
I got the quick filter idea from thunderbird, but I think it's close enough
![image](https://user-images.githubusercontent.com/31339007/102376127-a5c40300-4006-11eb-970d-e3e0a60fd8d2.png)

**Comment by austinwbest** [comment]
Datatable would resolve that easily but not so sure how well that would play with virtual tables

**Comment by planetrocky** [comment]
> > I don't understand why I have to create a filter each time I want to search a movie within lists in "Discover"
> 
> I'm not sure what you're asking?

I do!  Discover lists can return hundreds or thousands of films (Criterion Collection as one example).  Especially on a cell phone, it would be a lot easier to handle with a quick-search bar.
